spec/tools/fix_auth/auth_config_model_spec.rb

```diff
@@ -7,20 +7,17 @@
 require "fix_auth/models"
 
 describe FixAuth::AuthConfigModel do
+  let(:v1_key)  { MiqPassword.generate_symmetric }
   let(:pass)    { "password" }
   let(:enc_v1)  { MiqPassword.new.send(:encrypt_version_1, pass) }
-  let(:enc_v2)  { MiqPassword.new.send(:encrypt_version_2, pass) }
   let(:bad_v2)  { "v2:{5555555555555555555555==}" }
-  let(:enc_leg) { MiqPassword.v0_key.encrypt64(pass) }
 
   before do
-    MiqPassword.v0_key ||= CryptString.new(nil, "AES-128-CBC", "9999999999999999", "5555555555555555")
-    MiqPassword.v1_key ||= EzCrypto::Key.generate(:algorithm => "aes-256-cbc")
```

Comment:
What's this change?  Using the convenience methods (generate_symetric and add_legacy_key) instead of privates?  Why drop v0 key?  Not needed?